homeassistant/components/workday/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the Workday integration."""
+from homeassistant.const import WEEKDAYS
```

Comment:
Please try to keep the config-flow PR as small as possible.
For example, this constants file can be created in a preliminary PR.